draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -246,76 +312,106 @@ Internet-Draft      Network Slice Topology Data Model         March 2023
+   reserved and represented in the form of a customized topology, which
+   can then be mapped to a network resource partition (NRP) according to
+   the scenarios defined in [I-D.ietf-teas-ietf-network-slices].
+
+   Network slices may be abstracted differently depending on the
+   requirement of the network slice customer.  A customer may request a
+   network slice with direct connectivity between pairs of service
+   demarcation points (SDPs).  Each connection within the network slice
+   may be further supported by an end-to-end tunnel that traverse a
+   specific path in a given customized topology supplied by the
+   customer.  The resources associated with a link are immediately
+   commissioned by the realization at the time of network slice
+   configuration.
+
+   Alternatively, a customer can request resources to be reserved for
+   potential network slices through a customized topology, and use the
+   resources to build network slices in the future.  The customized
+   topology represents resources that are reserved but not commissioned
+   at the time of request.  By doing so, customers can share resources
+   between multiple endpoints and use them on demand.
+
+   In the example shown in Figure 2, two customized topology intents
+   named as Network Slice Blue and Network Slice Red, are created by
+   separate customers and delivered to the network slice service
+   provider.  The provider maps the two intents to corresponding network
+   resource partitions (NRPs) internally.  In realizing the network
+   resource partitions, node virtualization is used to separate and
+   allocate resources in physical devices.  Two virtual routers VR1 and
+   VR2 are created over physical router R1, and two virtual routers VR3
+   and VR4 are created over physical router R2, respectively.  Each of
+   the virtual routers,as a partition of the physical router, takes a
    portion of the resources such as ports and memory in the physical
    router.
    Depending on the requirements and the implementations, they may share
    certain resources such as processors, ASICs, and switch fabric.
 
-   The network slice topology intent requested by the customers is then
-   mapped to a corresponding network resource partition.  The provider
-   also reports the operational state of the topology, which shows the
-   resources that are allocated.  Customers can process the requested
-   topology and integrate it with their own topology.
+
+
+
+
+
+
+
+
+
+
+Liu, et al.             Expires 31 December 2023                [Page 7]
+
+Internet-Draft      Network Slice Topology Data Model          June 2023
+
+
+   A network slice customer can configure customized topologies without
+   prior knowledge of the provider's network and its resource
+   availability.  Alternatively, the provider may choose to expose its
+   resource availability through an abstract topology to the customer.
+   This enables the customer to understand and use this knowledge to
```

Comment:
I would say "This enables the customer to have the basic knowledge to build customized topologies."

Reply:
how about "This equips the customer with further knowledge to build customized topologies."

Reply:
Proposed text:
~~~
   A network slice customer can configure customized topologies without prior knowledge
   of the provider's network and its resource availability. Alternatively, the provider may
   choose to expose its resource availability through an abstract topology to the customer.
   This equips the customer with further knowledge to build customized topologies.
   For example, the customer could specify the nodes and links in a customized topology to
   share the same ids with, or reference, the corresponding nodes and links in the abstract 
   topology exposed a prior by the provider.
~~~